### DISCUSSION TITLE
Adding Assignee: All GitHub users are searched. Why?

### DISCUSSION THREAD

**Discussion by PhilippFeO** [pwntester/octo.nvim#575] [Q&A] [answered]
The title basically say it all. Why is the complete user base searched when adding an assignee and not just the maintainers/owners of a repository?

Can I limit the search to this group?

**Comment by williambdean** [comment, marked as answer]
I added some functionality recently which allows for repo specific assignable or mentionable users. 

Will be on the last master branch. Give it a try!

Details: https://github.com/pwntester/octo.nvim/pull/594

**Reply by PhilippFeO** [reply]
It work's. Great! :blush:

**Reply by williambdean** [reply]
Id also recommend the git-cmp which will use a similar list for tagging individuals

**Reply by PhilippFeO** [reply]
Are you referring to [petertriho/cmp-git](https://github.com/petertriho/cmp-git)? I use this for my Commit messages (occasionally). I can't imagine how it integrates into octo.nvim, especially due to the predefined very handy key mappings like `<Leader>aa`.

**Reply by williambdean** [reply]
Correct. The config in the README of that repo puts "@" as trigger for mentionable users in "octo" buffers which is nice when trying to tag users (different than assigning and adding to reviews via octo workflow)

**Reply by PhilippFeO** [reply]
Ah, nice to know for comments and descriptions!
